Community-Committee-Charter.md

```diff
@@ -24,7 +24,7 @@ Changes to CommComm membership should be posted in the agenda, and may be sugges
 
 No more than one-fourth of the CommComm members may be affiliated with the same employer or leadership of a community/ecosystem organization. If removal or resignation of a CommComm member, or a change of employment by a CommComm member, creates a situation where more than one-fourth of the CommComm membership shares an employer, then the situation must be immediately remedied by the resignation or removal of one or more CommComm members affiliated with the over-represented employer(s).
 
-The CommComm members shall consist of active members of Community Projects and the two Individual Membership Directors as defined in Section 10.
+The CommComm members shall consist of active members of Community Projects and the two Individual Membership Directors.
```

Comment:
We should remove this requirement that an Individual Membership Director is automatically a member. @williamkapke is active in CommComm as a member regardless and I think that's a much more organic approach to input and information gathering.